warehouse/templates/email/two-factor-mandate/body.html

```diff
@@ -0,0 +1,37 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+{% extends "email/_base/body.html" %}
+
+
+{% block content %}
+<p>
+Congratulations! A project you ('{{ username }}') maintain has been designated as a critical project.
```

Comment:
Needs `trans` markers here and below I think, for eventual i18n?

Reply:
These email templates join some of our other email templates that don't have a logged in user, thus we don't have a locale for them to translate this with, so it doesn't really make sense to mark these for translation now.